nepalingo-web/package.json

```diff
@@ -17,6 +17,7 @@
     "react-dom": "^18.3.1",
     "react-ga4": "^2.1.0",
     "react-router-dom": "^6.24.1",
+    "react-supabase": "^0.2.0",
```

Comment:
Are you still using react-supabase? You can run `pnpm remove react-supabase` to get rid of it if you don't need it anymore

https://chatgpt.com/share/04fc4457-0e8f-40f1-b135-e59fe0c94d16